components/paypal/sources/new-payment-event/new-payment-event.mjs

```diff
@@ -0,0 +1,40 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  name: "New Payment Event (Instant)",
+  version: "0.0.1",
+  key: "paypal-new-payment-event",
+  description: "Emit new event on an new payment webhook event. [See webhooks docs here](https://developer.paypal.com/docs/api/webhooks/v1/#webhooks_post) and [Events Types docs here](https://developer.paypal.com/docs/api/webhooks/v1/#webhooks-event-types_list)",
```

Comment:
```suggestion
  description: "Emits a new event on a new payment webhook event. [See webhooks docs here](https://developer.paypal.com/docs/api/webhooks/v1/#webhooks_post) and [Events Types docs here](https://developer.paypal.com/docs/api/webhooks/v1/#webhooks-event-types_list)"
```

Reply:
Changed 'Emit new event' to 'Emits a new event' to follow Google's Developer docs style guide (Use present tense and active voice)